Fixes Foreman Support #22332 - document multiple pxdns (#2007)
theforeman/foreman-documentation: guides/common/modules/con_unattended-use-customization-and-image-remastering.adoc

fdi.pxip, fdi.pxgw, fdi.pxdns::
Manually configures IP address (`fdi.pxip`), the gateway (`fdi.pxgw`), and the DNS (`fdi.pxdns`) for the primary network interface.
If you omit these parameters, the image uses DHCP to configure the network interface.

> You can add multiple DNS entries in a comma-separated list (for example `fdi.pxdns=192.168.1.1,192.168.200.1`

Reviewer: ```suggestion
You can add multiple DNS entries in a footnote:[comma-separated] list (for example `fdi.pxdns=192.168.1.1,192.168.200.1`
```
However, I would move the footnote closer to the culprit.

Author: agreed, applied as 0e097b2143b256d1c81cb26d644144d9fe10ccff

> footnote:[Please note that using `,` abuses the fact that NetworkManager expects a `;` but currently also accepts a `,`

Reviewer: ```suggestion
footnote:[NetworkManager expects `;` as list separators but currently also accepts `,`.
```

> see also `man nm-settings-keyfile` and <https://www.gnu.org/software/grub/manual/grub/grub.html#Shell_002dlike-scripting>]).

Reviewer: ```suggestion
For more information, see `man nm-settings-keyfile` and https://www.gnu.org/software/grub/manual/grub/grub.html#Shell_002dlike-scripting[Shell-like scripting in GRUB]]).
```

fdi.pxip, fdi.pxgw, fdi.pxdns::
Manually configures IP address (`fdi.pxip`), the gateway (`fdi.pxgw`), and the DNS (`fdi.pxdns`) for the primary network interface.
If you omit these parameters, the image uses DHCP to configure the network interface.
You can add multiple DNS entries in a comma-separated
footnote:[NetworkManager expects `;` as a list separator but currently also accepts `,`.
For more information, see `man nm-settings-keyfile` and https://www.gnu.org/software/grub/manual/grub/grub.html#Shell_002dlike-scripting[Shell-like scripting in GRUB]]

> list (for example `fdi.pxdns=192.168.1.1,192.168.200.1`).

Reviewer: ```suggestion
list, for example `fdi.pxdns=192.168.1.1,192.168.200.1`.
```
Easier to read.

fdi.pxip, fdi.pxgw, fdi.pxdns::
Manually configures IP address (`fdi.pxip`), the gateway (`fdi.pxgw`), and the DNS (`fdi.pxdns`) for the primary network interface.

> If your omit these parameters, the image uses DHCP to configure the network interface.

Reviewer 2: I know this is unrelated to your PR, but if possible, can you fix this typo:
```suggestion
If you omit these parameters, the image uses DHCP to configure the network interface.
```

> Separate multiple DNS with comma (for example `fdi.pxdns=192.168.1.1,192.168.200.1`).

Reviewer 2: ```suggestion
You can add multiple DNS entries in a comma-separated list (for example `fdi.pxdns=192.168.1.1,192.168.200.1`).
```
